packages/p2p/src/stores/my-ads-store.js

```diff
@@ -819,3 +867,11 @@ export default class MyAdsStore extends BaseStore {
         return errors;
     }
 }
+
+const checkForFixedRateAds = ads_list => {
+    return ads_list.some(ad => ad.rate_type === ad_type.FIXED);
+};
+
+const checkForFloatRateAds = ads_list => {
+    return ads_list.some(ad => ad.rate_type === ad_type.FLOAT);
+};
```

Comment:
This does not need to be a function. It is only used once which makes it hard for someone to find out what is it actually doing

Reply:
Will make this change